test-main.js

```diff
@@ -5,6 +5,7 @@ var TEST_REGEXP = /tests\/.*\.js$/i;
 var realSinon = sinon;
 beforeEach(function() {
     sinon = realSinon.sandbox.create();
+    localStorage.clear();
```

Comment:
The suite stays green. I was surprised we weren't doing this already. When I wrote the test I added it started failing after I'd run it once and this fixes it. We do this for our ui tests also. https://github.com/mozilla/marketplace-gulp/blob/master/tests/casper-helpers.js#L308